src/liquid/components/ld-label/ld-label.tsx

```diff
@@ -52,12 +68,7 @@ export class LdLabel {
     ])
 
     return (
-      <label
-        class={cl}
-        onClick={this.handleClick}
-        part="tag"
-        {...cloneAttributes(this.el, ['align-message', 'position', 'size'])}
-      >
+      <label class={cl} onClick={this.handleClick} part="tag">
```

Comment:
`{...this.clonedAttributes}` is missing.

Reply:
Good catch! Thanks! Fixed! 🤗